src/App.jsx

```diff
@@ -11,6 +12,7 @@ const App = () => {
       </header> */}
       <main className="main">
         <Outlet />
+        <Payment />
```

Comment:
to move it into routes and separate the item.

Reply:
done, payment component is in the routes folder